doc/index.md

```diff
@@ -456,6 +456,19 @@ Returns: a Promise that is fulled one the web socket is connected
   and other necessary async tasks are complete.
 
 
+#### `startSoftphoneSesssion(softphoneSessionParams: IStartSoftphoneSessionParams)`
```

Comment:
You have 4 `s`es in `Session`. Also a small NIT: I haven't been included the params in the function header since it is included in the declaration below.